articles/libraries/auth0js/v9/migration-guide.md

```diff
@@ -0,0 +1,60 @@
+---
+section: libraries
+title: Migrating to Auth0.js v9
+description: How to migrate to Auth0.js v9
+toc: true
+---
+# Migrating to Auth0.js v9
+
+[Auth0.js v9](/libraries/auth0js) has been improved to work better in [embedded login scenarios](/guides/login/centralized-vs-embedded). It operates with enhanced security and removes dependencies that have been deprecated as per Auth0's roadmap. In some cases, these security enhancements may impact application behavior when upgrading from an earlier versions of auth0.js. 
+
+## Should I migrate to v9?
+
+Auth0.js can be used to implement authentication in different ways:
+
+- In your application, to trigger centralized login, using the `.authorize()` method, where the user is redirected from your website to an Auth0's Hosted Login Page.
+
+- In your application, to implement embedded login, using the `.login()`, `popup.loginWithCredentials()`, `client.loginWithCredentials()` methods, where the login dialog is displayed in the application's website.
+
+- In the [Hosted Login Page](/hosted-pages/login), where you can use the same methods as in embedded login but from inside a customized Auth0's Hosted Login Page. Most customers don't customize Auth0 Hosted Login Page with auth0.js, so your probably don't need to worry about this scenario.
+
+Depending on how you are using auth0.js, you have different options:
+
+| **Scenario** | **Migration to v9** | 
+| --- | --- | 
+| In your application, to trigger centralized login | Recommended |
+| In your application, to implement embedded login | Required |
```

Comment:
How about

> Required (read below)

instead of just

> Required

to make sure the centralized login argument below is more emphasized?